classification/views/exports/classification_export_formatter_lab_compare.py

```diff
@@ -13,10 +15,14 @@
 
 class ClassificationlabCompareRow(ExportRow):
 
-    def __init__(self,  AlleleData, lab1, lab2):
+    def __init__(self,  AlleleData, lab1, lab2, comment=None, authorised_date=None, created_date=None, Patient_id=None):
```

Comment:
minor but have `patient_id` instead of `Patient_id`

Reply:
Likewise `AlleleData` should be `allele_data` or even better `allele_data:AlleleData`.
This is a good chance to start adding type hints to the parameters